example.yaml

```diff
@@ -33,11 +33,16 @@ http_address: "localhost:8127"
 forward_address: "http://veneur.example.com"
 
 ### TRACING
-# The address on which we will listen for trace data
+# The address on which we will listen for UDP trace data
 trace_address: "127.0.0.1:8128"
-# Use a static host to send traces to
+# Use a static host to send datadog traces to
```

Comment:
It actually... isn't. We need to update the names so the namespacing in our config is correct, but the static host is for the Datadog agent (formerly the Datadog tracing agent). I didn't really want to shave that yak in this pull request, so I left this comment as a psuedo-`TODO.` :)